python/run-tests.py

```diff
@@ -22,22 +22,28 @@
 from os import path
 
 
-def test(root_dir, package):
-    # Run all of the test under test/python directory, each of them
+def test(root_dir, test_path, package):
+    # Run all of the tests under test_path directory, each of them
     # has main entry point to execute, which is python's unittest testing
     # framework.
     python_root_dir = path.join(root_dir, "python")
-    test_dir = path.join(python_root_dir, path.join("delta", "tests"))
+    test_dir = path.join(python_root_dir, path.join(test_path, "tests"))
     test_files = [os.path.join(test_dir, f) for f in os.listdir(test_dir)
                   if os.path.isfile(os.path.join(test_dir, f)) and
                   f.endswith(".py") and not f.startswith("_")]
-    extra_class_path = path.join(python_root_dir, path.join("delta", "testing"))
+    extra_class_path = path.join(python_root_dir, path.join(test_path, "testing"))
 
     for test_file in test_files:
         try:
             cmd = ["spark-submit",
                    "--driver-class-path=%s" % extra_class_path,
-                   "--packages", package, test_file]
+                   "--packages", package +
+                   ",io.delta:delta-connect-server_2.13:4.0.0-SNAPSHOT," +
+                   "io.delta:delta-connect-common_2.13:4.0.0-SNAPSHOT," +
+                   "org.apache.spark:spark-connect_2.13:4.0.0-preview1," +
+                   "org.apache.spark:spark-connect-client-jvm_2.13:4.0.0-preview1," +
+                   "org.apache.spark:spark-connect-common_2.13:4.0.0-preview1," +
+                   "com.google.protobuf:protobuf-java:3.25.1", test_file]
```

Comment:
In the final product it shouldn't have to be the necessary to manually include this. It's also not necessary for Spark (and somehow also not for our Scala client tests?). Let's look at how Spark assembles its jars (including it's shading of this library, which we will also have to do).